codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/utils/Constants.java

```diff
@@ -0,0 +1,27 @@
+package software.amazon.polymorph.smithygo.utils;
+
+import software.amazon.smithy.model.shapes.MemberShape;
+
+public class Constants {
+    public static final String DAFNY_RUNTIME_GO_LIBRARY_MODULE = "github.com/dafny-lang/DafnyRuntimeGo/v4";
+
+    // TODO: Is it possible to make this function name shorter and in camelCase?
+    /**
+     * Generates a function name for shape visitors for AWS SDK and localservice.
+     * 
+     * @param memberShape The visiting MemberShape
+     * @param suffix A string to be appended at the end of the generated function name
+     * @return A string representing the generated function name
+     */
+    public static String funcNameGenerator(
+    final MemberShape memberShape,
+    final String suffix
+    ) {
+        return memberShape
+        .getId()
+        .toString()
+        .replaceAll("[.$#]", "_")
+        .concat("_")
+        .concat(suffix);
```

Comment:
Why is this different than one in the `ValidationGenerator` ?

Reply:
I thought the name in ValidationGenerator won't collide and will look good but the name collided (Caught in CI in [8738b87](https://github.com/smithy-lang/smithy-dafny/pull/641/commits/8738b875b4e1d6678bbcf7ee2c5d4736a050d2d9)) So, I made ValidationGenerator to use same function name.